ckanext/nhsengland/plugin.py

```diff
@@ -119,6 +119,7 @@ def before_map(self, route_map):
         """
         with routes.mapper.SubMapper(route_map,
             controller='ckanext.nhsengland.plugin:NHSEController') as m:
+            m.connect('dashboard', '/dash', action='dashboard')
```

Comment:
As mentioned in the email... how about we put this stuff under a `/dashboards` root..? So we get `/dashboards/board-report` as the second argument. Any subsequent reports can be `/dashboards/foo` etc etc etc